internal/campaigns/run_steps.go

```diff
@@ -97,6 +97,12 @@ func runSteps(ctx context.Context, wc *WorkspaceCreator, repo *graphql.Repositor
 		}
 		defer os.Remove(runScriptFile.Name())
 
+		// This file needs to be readable within the container regardless of
+		// the user the container is running as.
+		if err := runScriptFile.Chmod(0644); err != nil {
```

Comment:
does it not need to be executable?

Reply:
Counter intuitively, no! The Docker command basically boils down to this:

```sh
docker run --entrypoint /bin/bash -- sha256:CONTAINER_ID /tmp/some-horrible-script-name
```

Since the shell is the entrypoint, only that needs to be executable, and the script being run is just a regular old command line parameter.